symengine/tests/cwrapper/test_cwrapper.c

```diff
@@ -83,11 +83,31 @@ void test_basic() {
     basic_free_heap(y);
 }
 
+void test_complex() {
+    basic e;
+    basic f;
+    char* s;
+    basic_init(e);
+    basic_init(f);
```

Comment:
It works after `git clean -dfx`. I think I might have corrupted some include path somehow.

Thanks a lot @certik!